### PR TITLE
Created the ContentSectionWrapper, Created About Component, Added bot…

### PR DESCRIPTION
**Pull request by margarita-poole**
…h to the Home Page

### CODE REVIEW

File: components/atoms/ContentSectionWrapper/ContentSectionWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Children, FC } from 'react';
```
Comment (Earle-Poole) [inline components/atoms/ContentSectionWrapper/ContentSectionWrapper.tsx]: `Children` is unused in this import. Please remove.

File: components/templates/home/HomeTemplate.tsx [hunk 2]
```diff
@@ -9,26 +11,35 @@ const HomeTemplate = () => {
...
           <div className='flex mx-auto w-fit my-4'>
-            <input type='email' placeholder='email' className='w-80 text-black text-xl p-2'></input>
+            <input
+              type='email'
+              placeholder='email'
+              className='w-80 text-black text-xl p-2'
+            ></input>
             <MyButton href={urls.MAILCHIMP} name='Subscribe to Me' />
           </div>
         </div>
```
Comment (Earle-Poole) [inline components/templates/home/HomeTemplate.tsx]: Shouldnt this also be in a ContentSectionWrapper?

File: components/templates/home/HomeTemplate.tsx [hunk 3]
```diff
@@ -9,26 +11,35 @@ const HomeTemplate = () => {
     <div className='w-full flex flex-col items-center'>
       <CoverPhoto />
       <div className='w-full max-w-7xl flex flex-col items-center'>
+        <ContentSectionWrapper>
```
Comment (Earle-Poole) [inline components/templates/home/HomeTemplate.tsx]: Dont forget that we need to pass these a `background` prop

**Review by Earle-Poole — APPROVED** [review]
Approving this now, but I had some suggestions to bring this closer in line with requirements.

File: components/atoms/ContentSectionWrapper/ContentSectionWrapper.tsx (hunk 1 above)
Comment (margarita-poole) [reply components/atoms/ContentSectionWrapper/ContentSectionWrapper.tsx]: Good catch. It has been removed.

File: components/templates/home/HomeTemplate.tsx (hunk 2 above)
Comment (margarita-poole) [reply components/templates/home/HomeTemplate.tsx]: Totally right. It has been added.

File: components/templates/home/HomeTemplate.tsx (hunk 3 above)
Comment (margarita-poole) [reply components/templates/home/HomeTemplate.tsx]: Completed. Thanks

**Pull request merged**